test-cases/features/alternator-ttl/longevity-alternator-disable-enable-ttl.yaml

```diff
@@ -46,6 +46,11 @@ alternator_enforce_authorization: true
 alternator_access_key_id: 'alternator'
 alternator_secret_access_key: 'password'
 
+authenticator: 'PasswordAuthenticator'
+authenticator_user: cassandra
+authenticator_password: cassandra
```

Comment:
why we need to credentials, one this and the other from above: alternator_access_key_id ?

Reply:
alternator_access_key_id and alternator_secret_access_key, can be used when not using scylla, but actual dynamodb

Reply:
we could use the same, alternator shares the same credentials

Reply:
as I said, we can have cases we are not using alternator, and I would like to keep it like that for now

Reply:
yes you stated, but didn't explain why?

Reply:
so the code uses `alternator_secret_access_key: 'password'` for dynamoDB ?

Reply:
in case `alternator_enforce_authorization` isn't used, `alternator_secret_access_key` and `alternator_access_key_id` are used.

at the end we were working on getting the credentials straight to the node running ycsb without this, but it wasn't finished
https://github.com/scylladb/scylla-cluster-tests/pull/5171

Reply:
until we support it maybe we should not have this dual credentials because it's difficult to say if something is useful if it is not used